src/styles/index.scss

```diff
@@ -1,3 +1,273 @@
+  padding: 40px 16px 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__code {
+  width: 98px;
+  height: 14px;
+  padding: 4px 16px;
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+}
+
+.card:hover {
+  border: 1px solid blue;
+  border-radius: 5px;
+  transform: scale(1.2);
+}
+
+.card:hover .card__name {
+  color: #34568b;
+}
+
+.card__price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 24px 16px 0;
+}
+
+.card__price-key {
+  width: 31px;
+  height: 18px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+  color: #616070;
+}
+
+.card__price-value {
+  width: 50px;
+  height: 18px;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+  color: #060b35;
+}
+
+.review {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 0;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.review__reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star-active {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star-active.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  box-sizing: border-box;
+  align-items: center;
+  margin: 0 4px 0 0;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star-active:last-child {
+  margin-right: 0;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+}
+
+.button:hover {
+  background: #fff;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
+
+.card:hover .button {
+  /* background-color: #FFF; */
+
+  /* color: #00acdc; */
+  background: #fff;
+  border: 1px solid #00acdc;
+  color: #00acdc;
+}
+
+.main {
+  --column-count: 1;
+
+  width: 100%;
+  border: 3px solid black;
+  display: grid;
+
+  gap: 46px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+  grid-template-columns: repeat(var(--column-count), minmax(200px, 240px));
+
+  @media (min-width: 488px) {
+    --column-count: 2;
+  }
+
+  @media (min-width: 768px) {
+    --column-count: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --column-count: 4;
+  }
+}
+
+:hover {
+  transition-duration: 300ms;
+}
```

Comment:
The `:hover` selector here is not attached to any specific class or element. It's a global style that applies a transition duration to all hoverable elements. This can lead to unintended effects. It should be scoped to specific elements or classes where a hover transition is desired.